sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/models/FaultInjectionRuleBuilder.java

```diff
@@ -0,0 +1,131 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.cosmos.models;
```

Comment:
maybe I should put all the fault injection related builders into a separate package: faultinjection

Reply:
+1 too much clutter in the "normal" case due to FaultInjection types

Reply:
Moved all FaultInjection related models into new package faultinjection